Core/Core/CourseSync/CourseSyncSelector/Model/CourseSyncSelectorInteractor.swift

```diff
@@ -34,6 +34,7 @@ public protocol CourseSyncSelectorInteractor: AnyObject {
     func setCollapsed(selection: CourseEntrySelection, isCollapsed: Bool)
     func toggleAllCoursesSelection(isSelected: Bool)
     func getSelectedCourseEntries() -> AnyPublisher<[CourseSyncEntry], Never>
+    func getUnSelectedCourseIds() -> AnyPublisher<[String], Never>
```

Comment:
IMHO it should be Un`s`elected.
Or maybe rather `deselected`, to match the state, but I may be wrong about that.
If you update please update all derived properties, methods, etc.